packages/path_provider/path_provider_windows/lib/src/path_provider_windows_real.dart

```diff
@@ -47,6 +47,11 @@ class VersionInfoQuerier {
 ///
 /// This class implements the `package:path_provider` functionality for Windows.
 class PathProviderWindows extends PathProviderPlatform {
+  /// Registers the Windows implementation.
+  static void registerWith() {
+    // TODO(stuartmorgan): Fill this up.
```

Comment:
It's just like the Linux version

Reply:
do you mean, it's not needed for windows?

Reply:
fwiw, pubspec.yaml says the opposite: https://github.com/flutter/plugins/blob/1a1ac3eab74be0c579168cf7680218af07420365/packages/path_provider/path_provider_windows/pubspec.yaml#L11

Reply:
Just like the Linux version meaning it should set itself as the default instance:
```
PathProviderPlatform.instance = PathProviderWindows();
```

Reply:
done